config/application.rb

```diff
@@ -1,35 +1,19 @@
 require_relative 'boot'
 
 require 'rails/all'
-require_relative '../app/middlewares/rack/domain_redirect'
```

Comment:
i am guessing we still need all of these

Reply:
Yep!